Userland/Libraries/LibIMAP/Objects.h

```diff
@@ -18,11 +18,38 @@
 namespace IMAP {
 enum class CommandType {
     Capability,
+    List,
     Noop,
+    Select,
+};
+
+enum class MailboxFlag : unsigned {
+    SAll = 1u << 0,
```

Comment:
SWhy sdo sthese snames sall sstart swith ss except that last one?

Reply:
In the spec they're all prefixed with a `\` so I prefixed all of them with an `S` (the `Unknown` one is for unrecognized extension flags). If that's weird I'll get rid of the `S`.

Reply:
I don't think the `S` is necessary, `MailboxFlag::All` seems to be unambiguous.